src/app/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-breadcrumbs',
+  templateUrl: './breadcrumbs.component.html',
+  styleUrls: ['./breadcrumbs.component.scss']
+})
+export class BreadcrumbsComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
```

Comment:
let's make a convention to remove an empty constructor and LC hook

Reply:
done